sdcm/nemesis.py

```diff
@@ -3142,6 +3174,31 @@ def disrupt(self):
         self.disrupt_remove_node_then_add_node()
 
 
+class ScyllaOperatorGrowShrinkClusterNemesis(GrowShrinkClusterNemesis):
+    kubernetes = True
+
+    def set_target_node(self):
+        self.target_node = self.cluster.nodes[-1]  # can withdraw last node only
+        self.set_current_running_nemesis(node=self.target_node)
+        self.log.info('Current Target: %s with running nemesis: %s', self.target_node, self.target_node.running_nemesis)
+
+    @log_time_elapsed_and_status
+    def disrupt(self):
+        self.disrupt_grow_shrink_cluster()
+
```

Comment:
Weird, I don't know how this got here... probably something with a recent reabse.